LICENSE.txt

```diff
@@ -187,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
```

Comment:
I think we should also add their copyrights even though they didn;t do this in their Repo :thinking: 

```
Copyright 2021 Software Mansion

Copyright (C) 2002-2021 ProcessOne, SARL. All Rights Reserved.
```